content/docs/rfcs/1/README.md

```diff
@@ -48,8 +48,10 @@ if not, see http://www.gnu.org/licenses.
 This document is governed by the [1/COSS](spec/1) (COSS).
 
 ## Language
+If keywords are used in the RFC2119 format,
```

Comment:
Add for better clarity of use, https://github.com/vacp2p/rfc/pull/642#discussion_r1438099267_